ext/opcache/shared_alloc_mmap.c

```diff
@@ -87,6 +91,12 @@ static int create_segments(size_t requested_size, zend_shared_segment ***shared_
 	if (p != MAP_FAILED) {
 		goto success;
 	}
+#elif defined(VM_MAKE_TAG)
+	/* allows tracking segments via tools such as vmmap */
+	p = mmap(NULL, requested_size, flags, MAP_SHARED|MAP_ANONYMOUS, VM_MAKE_TAG(251), 0);
```

Comment:
I think it would be better to use something like

```
int fd = -1;
#if defined(VM_MAKE_TAG)
fd = VM_MAKE_TAG(251)
#endif
```
and then use `fd` in the mmap calls. This makes sure this code continues working even if macos supports MAP_32BIT or MAP_HUGETLB in the future.